tests/helpers/test_condition.py

```diff
@@ -992,6 +992,79 @@ async def test_time_using_input_datetime(hass: HomeAssistant) -> None:
         condition.time(hass, before="input_datetime.not_existing")
 
 
+async def test_time_using_time(hass: HomeAssistant) -> None:
+    """Test time conditions using time entities."""
+    hass.states.async_set(
+        "time.am",
+        "2021-06-03 13:00:00.000000+00:00",  # 6 am local time
```

Comment:
These tests are not valid, as the state for time entities are not matching.

A `time` entity only contains a time, and not a full timestamp.

../Frenck